### DISCUSSION TITLE
Hope to be able to implement search function

### DISCUSSION THREAD

**Discussion by lxxself** [coddingtonbear/obsidian-web#145] [Ideas]
Show content when you search on popular search engines via your keywords，like [EINDEX/logseq-copilot](https://github.com/EINDEX/logseq-copilot)

**Comment by coddingtonbear** [comment]
I'm afraid I think that such functionality should probably be in its own dedicated chrome extension instead of squeezed into this one; so I'd need a lot more convincing before considering accepting a PR adding similar functionality here.
